client/src/components/Explorer/reducers/explorer.test.js

```diff
@@ -20,13 +20,7 @@ describe('explorer', () => {
     expect(explorer(initialState, { type: 'CLOSE_EXPLORER' })).toEqual(initialState);
   });
 
-  it('PUSH_PAGE', () => {
+  it('GOTO_PAGE', () => {
     expect(explorer(initialState, { type: 'PUSH_PAGE', payload: { id: 100 } })).toMatchSnapshot();
```

Comment:
```suggestion
    expect(explorer(initialState, { type: 'GOTO_PAGE', payload: { id: 100 } })).toMatchSnapshot();
```

You’ll also need to add the `transition` I assume.